import_export/widgets.py

```diff
@@ -371,10 +371,12 @@ class Meta:
 
     :param model: The Model the ForeignKey refers to (required).
     :param field: A field on the related model used for looking up a particular object.
+    :param use_natural_foreign_keys: Boolean for whether to use Django natural key functions, default False
     """
-    def __init__(self, model, field='pk', *args, **kwargs):
+    def __init__(self, model, field='pk', use_natural_foreign_keys=False, *args, **kwargs):
```

Comment:
I think we should remove `*args` here - it serves no purpose (I know you didn't add it).  The first sentence of the docstring comment is inaccurate as well (it's been like that for a while).  It would be great to correct the docstring.

Reply:
Happy to do both of those things.

Reply:
Actually I think the args thing may want to wait for someone else to look at it, as since it calls to super() with *args and **kwargs. Not sure if there could be any unintended consequences. 

For now I've just fixed the docstring which I am sure of :)

Reply:
yes, we'd have to arrange it as follows for it to work:

```
    def __init__(self, model, *args, field='pk', use_natural_foreign_keys=False, **kwargs):
```

but ok to leave that one for now.

Reply:
Actually the *args thing leaves me a little doubtful. It feels like there might be something I am missing there -- I see that the super() call passes *args, which could theoretically cause some interface weirdness down the line. It's kind of weird as is with args after the keyword arguments, which causes a lot of problems (You can't just reorganize *args because lots of other functions treat 'field' like a positional argument). 

It does pass tests without *args.